FizzBuzz/README.md

```diff
@@ -0,0 +1,19 @@
+FizzBuzz.rb
+==============
+
+programmer: Amanda Kom (no collaboration)
+--------------
+
+- FizzBuzz first checks to see if there are any multiples of both 3 and 5, which will print "FizzBuzz"
+- Next, it checks to see if the number is just a multiple of 3, which would print "Fizz"
+- If it's not a multiple of 3, it will check if the number is a multiple of 5, which would print "Buzz"
+- Finally, if the number is neither a multiple of 3 nor a multiple of 5, it will print out the input number
+-Added changes from @Brookr
+
+FizzBuzzAdvanced.rb
+==============
+
+programmer: Amanda Kom (collaboration with David Ker)
+--------------
+
+- FizzBuzz first takes key-value pairs from the client to indicate outputs
```

Comment:
This readme could be more descriptive.  You could explain how a user can use the set method to add key-value pairs, and then use the final method to see what the output will be.